rust/src/transformations/make_stable_expr/expr_len/test.rs

```diff
@@ -0,0 +1,72 @@
+use polars::df;
+use polars_plan::dsl::all;
+
+use crate::{
+    metrics::{InsertDeleteDistance, L2Distance, PartitionDistance},
+    transformations::{test_helper::get_test_data, StableExpr},
+};
+
+use super::*;
+
+#[test]
+fn test_select_make_expr_len() -> Fallible<()> {
+    let (lf_domain, lf) = get_test_data()?;
+    let expr_domain = lf_domain.aggregate(["chunk_2_bool", "cycle_5_alpha"]);
+
+    let t_sum: Transformation<_, _, _, L2Distance<f64>> =
+        len().make_stable(expr_domain, PartitionDistance(InsertDeleteDistance))?;
+    let expr_res = t_sum.invoke(&(lf.logical_plan.clone(), all()))?.1;
+    assert_eq!(expr_res, len());
```

Comment:
This is coming from `super::*`? Maybe drop that and add `polars_plan::dsl::len` so it's easier for the reader to know where names come from.